packages/apps/reputation-oracle/server/src/modules/nda/nda.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+import { NDAController } from './nda.controller';
+import { NDAService } from './nda.service';
+import { UserModule } from '../user/user.module';
+import { UserRepository } from '../user/user.repository';
+
+@Module({
+  imports: [UserModule],
+  controllers: [NDAController],
+  providers: [NDAService, UserRepository],
```

Comment:
```suggestion
  providers: [NDAService],
```
`UserRepository` is provider by `UserModule` import